apps/bttester/syscfg.yml

```diff
@@ -33,11 +33,11 @@ syscfg.defs:
 
     BTTESTER_RTT_BUFFER_SIZE_UP:
         description: Bttester upstream buffer size
-        value: 512
+        value: MYNEWT_VAL_BTTESTER_BTP_DATA_SIZE_MAX
```

Comment:
should be `MYNEWT_VAL(BTTESTER_BTP_DATA_SIZE_MAX)` so newt can evaluate it

Reply:
Thanks, fixed.